src/utils/ConfigHelper.ts

```diff
@@ -141,6 +141,57 @@ const configs: ConfigHelperConfig[] = [
     dispenserAddress: null,
     metadataContractAddress: null,
     startBlock: 90707
+  },
+  {
+    networkId: 2021000,
+    network: 'gaiaxtestnet',
+    nodeUri: 'https://gaia-x.rpc',
+    metadataCacheUri: 'https://aquarius.gaiaxtestnet.oceanprotocol.com',
+    providerUri: 'https://provider.gaiaxtestnet.oceanprotocol.com',
+    subgraphUri: 'https://subgraph.gaiaxtestnet.oceanprotocol.com',
+    explorerUri: 'https://gaiaxtestnet.explorer',
+    oceanTokenAddress: null,
+    oceanTokenSymbol: 'OCEAN',
+    factoryAddress: null,
+    poolFactoryAddress: null,
+    fixedRateExchangeAddress: null,
+    dispenserAddress: null,
+    metadataContractAddress: null,
+    startBlock: 0
+  },
+  {
+    networkId: 80001,
+    network: 'mumbai',
+    nodeUri: 'https://polygon-mumbai.infura.io/v3/',
+    metadataCacheUri: 'https://aquarius.mumbai.oceanprotocol.com',
+    providerUri: 'https://provider.mumbai.oceanprotocol.com',
+    subgraphUri: 'https://subgraph.mumbai.oceanprotocol.com',
+    explorerUri: 'https://explorer-mumbai.maticvigil.com/',
+    oceanTokenAddress: null,
+    oceanTokenSymbol: 'OCEAN',
+    factoryAddress: null,
+    poolFactoryAddress: null,
+    fixedRateExchangeAddress: null,
+    dispenserAddress: null,
+    metadataContractAddress: null,
+    startBlock: 0
+  },
+  {
+    networkId: 56,
+    network: 'bsc',
+    nodeUri: 'https://bsc-dataseed.binance.org/',
+    metadataCacheUri: 'https://aquarius.bsc.oceanprotocol.com',
+    providerUri: 'https://provider.bsc.oceanprotocol.com',
+    subgraphUri: 'https://subgraph.bsc.oceanprotocol.com',
+    explorerUri: 'https://bscscan.com/',
+    oceanTokenAddress: null,
+    oceanTokenSymbol: 'OCEAN',
```

Comment:
just a note that this might get a different symbol

Reply:
nop, it's OCEAN  (see https://bscscan.com/token/0xdce07662ca8ebc241316a15b611c89711414dd1a)